src/ui/src/app/shared/services/charts.service.ts

```diff
@@ -127,6 +127,19 @@ export class ChartsService {
       .catch(this.handleError);
   }
 
+  /**
+   * Get the URL for retrieving the chart's icon
+   * 
+   * @param {string} chart Chart object
```

Comment:
How does this annotation work? {string} is the returned value? What's the name of this annotation?

Reply:
It has no semantic meaning, but good catch, that should actually be the type of the parameter which is a Chart.